### PR TITLE
Upgrade 025

### PR DESCRIPTION
**Pull request by entantoencuanto**
WIP to upgrade to a version compatible with Decidim 0.25

### CODE REVIEW

**Review by microstudi — COMMENTED** [review]
Thanks for starting this, it looks amazing!

Just to facilitate things here, we won't maintain compatibility with version 0.24 due the nature of the webpacker change in 0.25, I think is not worth the effort.
So, we can safely remove any "legacy" reference (currently pointing to 0.23). However, let's maintain the `v25` partials folder for future releases 2 versions compatibility to be introduced again.

I'd might be commenting on this PR from time to time, or, if you prefer, we can set up a branch directly on this repository and I'll give permissions to you to write so both of us can work on this PR. What you think?

File: lib/decidim/decidim_awesome/version.rb [hunk 1]
```diff
@@ -4,6 +4,6 @@ module Decidim
   # This holds the decidim-decidim_awesome version.
   module DecidimAwesome
     VERSION = "0.7.2"
-    COMPAT_DECIDIM_VERSION = [">= 0.23.0", "< 0.25"].freeze
+    COMPAT_DECIDIM_VERSION = [">= 0.23.0", "< 0.26"].freeze
```
Comment (microstudi) [inline lib/decidim/decidim_awesome/version.rb:7]:
```suggestion
    COMPAT_DECIDIM_VERSION = [">= 0.25.0", "< 0.26"].freeze
```
I think this should be the version supported for the next version of awesome. Maintaining sprockets compatibility seems to much for what's is worth.

**Comment by entantoencuanto** [comment]
Thanks! At the moment the PR is in a fairly preliminary state, but all indications are appreciated.

I think it's a good idea that we can work in the same branch, so go ahead with moving it to Platoniq. It would also be good to give permissions to @ferblape, which can be of great help in the webpacker migration part

**Comment by microstudi** [comment]
@entantoencuanto I've given you permissions to write on this repo (also welcomed @ferblape). Please create the new PR yourself as I'd like to maintain your authorship!

**Comment by entantoencuanto** [comment]
Closed in favor of #148

**Pull request closed without merging**